riff-raff.yaml

```diff
@@ -23,7 +23,7 @@ templates:
     contentDirectory: cloudwatch-logs-management
     parameters:
       bucketSsmLookup: true
-      fileName: lambda.zip
+      fileName: cloudwatch-logs-management.zip
```

Comment:
`@guardian/node-riffraff-artifact` (sensibly IMO!) will always name a file `action/action.zip` as described in the  `artifact.json` file.

Here we bring `riff-raff.yaml` inline with this naming.